tests/incident/test_pdf_form_input.py

```diff
@@ -14,8 +13,13 @@ def test_url():
     return "http://foersom.com/net/HowTo/data/OoPdfFormExample.pdf"
 
 
+@pytest.mark.incident
 @pytest.mark.pynput
 def test_pdf_form_fill(driver, test_url):
+    try:
+        from pynput.keyboard import Controller, Key
+    except:
+        pytest.skip("Could not load pynput")
```

Comment:
So this is in here to prevent problems with pynput and headless mode. We haven't figured out how to get those two things to work together....so, that's future work.